CONTRIBUTING.md

```diff
@@ -473,7 +476,12 @@ missing a difficulty rating, and you should feel free to add one.
  - **E-needs-mvce** This bug is missing a minimal complete and verifiable
    example.
 
-**Module.** A more fine groaned categorization than area.
+(I don't know why it's called `E-`. Many issues are missing a difficulty rating,
+and you should feel free to add one.)
```

Comment:
The reason it is called `E-` is that the main Rust repo calls the same thing `E-`. As for why _they_ call it that, no idea. Anyway, I'd like to see this reworded without the use of "I", and also the parentheses can be removed.

Reply:
Yeah the "E-" might be for (E)asy but then E-hard doesn't make a lot of sense? "E-" for (E)ffort? Anyway that isn't my comment, its just moved down from its prior location _above_ the more important points of this section. Also as a peasant-contributor I find the "feel free" phrase kind of funny? (I don't have permission to add labels.) Care to re-word it as you like for inclusion here?

Reply:
BTW: There are some bots like  "bors" I think that allow peasants to suggest labels for there issues/PRs. This is why I was suggesting that earlier on discord.

Reply:
How about this?

> The name `E-` was taken from the labels used in the main Rust repository. Some issues are missing a difficulty rating, but feel free to ask on our Discord server if you want to know how difficult an issue is.

Reply:
Done with minor additional edits in 8abeaec, thanks.